src/main/java/no/fint/betaling/model/OrderItem.java

```diff
@@ -5,13 +5,14 @@
 import java.net.URI;
 
 @Data
-public class OrderLine {
-    private URI itemUri;
+public class OrderItem {
+    private String code;
```

Comment:
Is this the code of `Lineitem` ?  If so, it should perhaps be called `itemCode`.